card/mod/05_standard/set/all/rich_html/editing.rb

```diff
@@ -131,57 +141,70 @@ def default_edit_name_args args
       card:     { update_referencers: false }
     )
     args[:optional_toolbar] ||= :show
-    args[:buttons] = %{
-      #{ button_tag 'Rename and Update', disable_with: 'Renaming', class: 'renamer-updater', situation: 'primary' }
-      #{ button_tag 'Rename',            disable_with: 'Renaming', class: 'renamer'         }
-      #{ button_tag 'Cancel', class: 'slotter',  type: 'button', href: path }
-    }
-
+    args[:buttons] =
+      %{
+        #{submit_button text: 'Rename and Update', disable_with: 'Renaming',
+                        class: 'renamer-updater'}
+        #{button_tag 'Rename',
+                     data: { disable_with: 'Renaming' },
+                     class: 'renamer'}
+        #{button_tag 'Cancel', class: 'slotter',  type: 'button', href: path}
+      }
   end
 
-
   view :edit_type, perms: :update do |args|
     frame_and_form :update, args do
-    #'main-success'=>'REDIRECT: _self', # adding this back in would make main cards redirect on cardtype changes
+    # 'main-success'=>'REDIRECT: _self', # adding this back in would make main cards redirect on cardtype changes
```

Comment:
Line is too long. [113/80]<br>Incorrect indentation detected (column 4 instead of 6).